metric_learn/__init__.py

```diff
@@ -1,9 +1,14 @@
 from __future__ import absolute_import
 
 from .itml import ITML
+from .itml import ITML_Supervised
```

Comment:
Here and below, I'd put these into one import statement:

    from .itml import ITML, ITML_Supervised